application-services/custom/camera-management/appcamera/evam.go

```diff
@@ -218,6 +221,66 @@ func (app *CameraManagementApp) getPipelineStatus(deviceName string) (interface{
 	return nil, nil
 }
 
+// processEdgeXEvent is the function that is called when an EdgeX event is received
+
+func (app *CameraManagementApp) processEdgeXEvent(_ interfaces.AppFunctionContext, data interface{}) (bool, interface{}) {
+	if data == nil {
+		app.lc.Errorf("processEdgeXEvent: was called without any data")
+		return false, nil
+	}
+
+	if app.config.AppCustom.DefaultPipelineName == "" || app.config.AppCustom.DefaultPipelineVersion == "" {
+		app.lc.Errorf("processEdgeXEvent: PipelineName or PipelineVersion is empty")
```

Comment:
Change to warning or info. Wording should look more like: no default pipeline name/version specified, skip starting pipeline for device %s, or something like that.